### PR TITLE
Fix "Could not find or access"

### PR DESCRIPTION
**Pull request by gjedeer**
```
TASK [rust : Unarchive Rust] *******************************************************************************************************************************************************************
fatal: [79.133.48.245]: FAILED! => {"changed": false, "msg": "Could not find or access '/opt/rust/rust-1.23.0-x86_64-unknown-linux-gnu.tar.gz'"}
        to retry, use: --limit @/home/gdr/src/.../worker.retry
```

The reason is:

https://stackoverflow.com/questions/32698529/ansible-unarchive-input-file-not-found#32702447

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@
     src: "{{ rust_dir }}/{{ rust_archive_name }}"
     dest: "{{ rust_dir }}"
     creates: "{{ rust_archive_unpack_done }}"
+    remote_src: yes
```
Comment (ruslo) [inline tasks/main.yml:36]: Available since `2.2`, need to update `meta/main.yml` file.

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by ruslo** [comment]
Tests failed. I think it's not related to your pull request but if you know the fix feel free to share solution.

**Review by ruslo — APPROVED** [review]

**Comment by gjedeer** [comment]
Re. tests: it seems outbound network connections from the Travis container were down. I've had this in my projects too, it's random, you don't control it. Just look at the failed apt-get on the top.

**Comment by ruslo** [comment]
> it's random, you don't control it

I think it's pretty stable, something wrong with certificates. May be different Python version needed:
```
Failed to validate the SSL certificate for static.rust-lang.org:443. Make sure
your managed systems have a valid CA certificate installed. If the website
serving the url uses SNI you need python >= 2.7.9 on your managed machine
(the python executable used (/usr/bin/python) is version: 2.7.6 (default, Nov
23 2017, 15:49:48) [GCC 4.8.4]) or you can install the `urllib3`, `pyOpenSSL`,
`ndg-httpsclient`, and `pyasn1` python modules to perform SNI verification in
python >= 2.6. You can use validate_certs=False if you do not need to confirm
the servers identity but this is unsafe and not recommended.  Paths checked
for this platform: /etc/ssl/certs, /etc/pki/ca-trust/extracted/pem,
/etc/pki/tls/certs, /usr/share/ca-certificates/cacert.org, /etc/ansible. The
exception msg was: [Errno 1] _ssl.c:510: error:14077410:SSL
routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure.

```

**Comment by ruslo** [comment]
https://github.com/ruslo/ansible-rust/releases/tag/v1.0.2

**Comment by gjedeer** [comment]
Thanks for merging (and I stand by my diagnosis)

**Pull request merged**